extensions-core/kafka-indexing-service/pom.xml

```diff
@@ -67,7 +67,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.11</artifactId>
-      <version>0.9.0.0</version>
+      <version>0.10.0.0</version>
```

Comment:
@gianm Thanks! I will  update Kafka indexing service document and will also check with Kafka 0.10.2.0